src/test/java/http_server/ServerTest.java

```diff
@@ -24,16 +26,10 @@ private String runSessionAndRetrieveResponse(String request) {
     }
 
     @BeforeEach
-    public void prepFiles() {
-        FileHandler.deleteDirectory("./public/dog");
-        FileHandler.deleteDirectory("./public/cat");
-        FileHandler.deleteDirectory("./public/delete_me.txt");
-    }
-
-    @AfterEach
     public void cleanUpFiles() {
```

Comment:
Do you think this is actually required before every test unit? Do you think you can change how the MockRepository is implemented so that this cleanUp step is unnecessary?

Reply:
I've adjusted it to run like so now:
```
 @BeforeEach
    public void cleanRouter() {
        Repository mockRepository = new MockRepository("/public");
        mockRepository.writeFile("./public/Home.html", MIMETypes.html, "<!DOCTYPE html>\n".getBytes());
        mockRepository.writeFile("./public/TurtleTab.txt", MIMETypes.plain, "TurtleTabs a Google".getBytes());
        mockRepository.writeFile("./public/water.png", MIMETypes.png, "water image".getBytes());
        mockRepository.writeFile("./public/japan.png", MIMETypes.png, "japan image".getBytes());
        router = new MockRouter(mockRepository).getApp();
    }
```

Reply:
Creates a fresh repository before each test